### PR TITLE
Quirks for Smartthings MotionV4

### PR DESCRIPTION
**Pull request by roblandry**

### CODE REVIEW

**Comment by roblandry** [comment]
It actually looks like this matches the signatures for the CentraLite 3325-S and 3326-L. Being a different manufacturer im not sure if this will conflict like #19

File: zhaquirks/smartthings/motionv4.py [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    PollControl, Ota, BinaryInput
+from zigpy.zcl.clusters.security import IasZone
+from zhaquirks.centralite import PowerConfigurationCluster
+from zigpy.zcl.clusters.measurement import TemperatureMeasurement
+from zigpy.quirks import CustomDevice
+
+
+class SmartThingsMotionV4(CustomDevice):
+
+    def __init__(self, *args, **kwargs):
```
Comment (dmulcahey) [inline zhaquirks/smartthings/motionv4.py]: unneeded can remove the entire function

File: zhaquirks/smartthings/motionv4.py [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+        super().__init__(*args, **kwargs)
+
+    signature = {
+        #  <SimpleDescriptor endpoint=1 profile=260 device_type=1026
+        #  device_version=0
+        #  input_clusters=[0, 1, 3, 15, 1026, 1280, 32]
+        #  output_clusters=[25]>
+        1: {
+            'profile_id': zha.PROFILE_ID,
```
Comment (dmulcahey) [inline zhaquirks/smartthings/motionv4.py]: we match on man / model now so add them here

File: zhaquirks/smartthings/motionv4.py (hunk 1 above)
Comment (roblandry) [reply zhaquirks/smartthings/motionv4.py]: Done

File: zhaquirks/smartthings/motionv4.py (hunk 2 above)
Comment (roblandry) [reply zhaquirks/smartthings/motionv4.py]: Done

**Pull request merged**